config/linux-linuxboot.config

```diff
@@ -163,7 +163,7 @@ CONFIG_E1000E=m
 CONFIG_IGB=m
 # CONFIG_NET_VENDOR_I825XX is not set
 # CONFIG_NET_VENDOR_MARVELL is not set
-CONFIG_MLX4_EN=m
+CONFIG_MLX4_EN=y
```

Comment:
MLX4 should stay a module as well; not every platform needs it (and it's huge).

Reply:
I agree, one of the issue I face, is that in space constraint environment like winterfell, if we want support pxe boot with u-root, we do have to support mlx4 driver, and the blob is getting to fat to fit as a kernel module + elvish + pxeboot code in u-root. Is there a way to move such specific kernel configuration at the board level definition and perhaps concatenate these specific needs to build final kernel config ?